README.md

```diff
@@ -1 +1,124 @@
-Test
\ No newline at end of file
+# Trustly Parliament Challenge
+
+## Problem
+
+This is a challenge by Trustly known as the "Parliament Challenge".
+
+The Swedish parliament has an open API platform for developers to use its resources to create applications. Among its resources are the speeches in the parliament and the data of its members.
+
+The challenge by Trustly is to create an appliction that serves requests for parliament speeches. The expected response should be created by merging the data from the speeches api and members api and should be in a JSON format.
+
+The expected response should at least contain:
+
+- A unique ID, representing the speech itself (anforande_id)
+- Date of speech (dok_datum)
+- The name of the speaker, and only the name (tilltalsnamn)
+- Political affiliation (parti)
+- The official e-mail address (uppgift)
+- Constituency (valkrets)
+- A decent sized image that could be used in a web site listing (bild_url_192)
+- The debate subject (avsnittsrubrik)
+- A link to the speech (HATEOAS style) (protokoll_url_www)
+
+The link to the challenge can be found on https://github.com/trustly/parliament-challenge
+
+## Approach
+
+The challenge is approached as follows:
+
+1. The function `get_speeches()` is responsible for getting the speeches resources from its respective apis. The speeches api has a filter to extract the ten latest speeches. Since the data is in a nested dict, the for loop has been used to iterate over to the required dict which is "anforande". Speech api: http://data.riksdagen.se/anforandelista/?anftyp=Nej&sz=10&utformat=json
```

Comment:
Because it's in a nested dict, it doesn't mean you need to use the for loop.

Reply:
looped through relevant key to get speeches keys and values.